### PR TITLE
Chompsession model

### PR DESCRIPTION
**Pull request by tjw0612**
## Why

This pull request is needed because:

Created chompsession model
## What

This pull request introduces the following:

 * `Changes introduced`

`Add Implementation details if needed`

### Screenshot

`Attach any screenshots here. Remove when unneeded`

## How to Test

`Give the reviewer(s) some clues on how to test your PR`

## Other

`This section is for things that can't be put into the previous sections. Remove when unneeded`

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,11 +10,25 @@
...
+ActiveRecord::Schema.define(version: 2021_09_21_082544) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "chomp_sessions", force: :cascade do |t|
+    t.string "name"
+    t.datetime "date"
+    t.string "unique_identifier"
+    t.string "status"
```
Comment (mxng) [inline db/schema.rb:22]: @TeeWhyKay actually should we just make the Status into Completed and let it be boolean True or False?

**Review by mxng — COMMENTED** [review]
@TeeWhyKay @tjw0612 What do you guys think about instead of Status we use Completed as the column name and let the values be either True or False?

File: db/schema.rb (hunk 1 above)
Comment (tjw0612) [reply db/schema.rb:22]: I think should still stick to string, just in case, we want to account for:
E.g. 
"pending"
"completed"
"failed"

**Review by mxng — APPROVED** [review]
👍 👍

**Pull request merged**